charts/redpanda/values.go

```diff
@@ -157,6 +157,7 @@ type Monitoring struct {
 	ScrapeInterval string            `json:"scrapeInterval" jsonschema:"required,pattern=.*[smh]$"`
 	Labels         map[string]string `json:"labels"`
 	TLSConfig      map[string]any    `json:"tlsConfig"`
+	EnableHttp2    *bool             `json:"enabled" jsonschema:"required"`
```

Comment:
The json name of this field feels _very_ misleading to me.

Reply:
ahh thats me typing too fast and changing context, ill fix it.